src/createServer.js

```diff
@@ -1,3 +1,61 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable no-console */
+const { convertToCase } = require('./convertToCase');
+const { ErrorMessages } = require('./utils/ErrorMessages');
+const http = require('http');
+
+const createServer = () => {
+  return http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const { url } = req;
+    const urlSplit = url.split('?');
+    const params = new URLSearchParams(urlSplit[1]);
+    const textToConvert = urlSplit[0].slice(1);
+    const toCase = params.get('toCase');
+
+    const caseNames = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
```

Comment:
Better to move it outside this function, it's a static constant, so best option is to move it to separate file (at least outside function)